spec/hand_spec.rb

```diff
@@ -80,8 +80,67 @@
       hand.add_card(card2)
       hand.add_card(high_card)
       hand.add_card(card3)
-      result = hand.get_high_card
+      result = hand.high_card
       expect(result).to eq(high_card)
     end
   end
 end
+
+RSpec.describe Hand, '#contains_pair?' do
+  context 'When the hand contains 2 cards with the same value' do
+    it 'returns true' do
+      hand = Hand.new
+      card1 = Card.new('C', 4)
+      card2 = Card.new('H', 4)
+      hand.add_card(card1)
+      hand.add_card(card2)
+      result = hand.contains_pair?
+      expect(result).to be true
+    end
+  end
+  context 'When the hand does not contain 2 cards with the same value' do
+    it 'returns false' do
+      hand = Hand.new
+      card1 = Card.new('D', 6)
+      card2 = Card.new('H', 4)
+      hand.add_card(card1)
+      hand.add_card(card2)
+      result = hand.contains_pair?
+      expect(result).to be false
+    end
+  end
+end
+
+RSpec.describe Hand, '#value_hash' do
+  context 'When the hand contains 2 cards with the same value' do
+    it 'returns a hash with the value as the key and the frequency as 2' do
+      hand = Hand.new
+      card1 = Card.new('C', 4)
+      card2 = Card.new('H', 4)
+      hand.add_card(card1)
+      hand.add_card(card2)
+      result = hand.value_hash
+      value_hash = { 4 => 2 }
+      expect(result).to eq(value_hash)
+    end
+  end
+end
+
+RSpec.describe Hand, '#value_hash' do
+  context 'When the hand contains 2 pairs of cards with the same value' do
+    it 'returns a hash both pairs as the key and the frequency as 2' do
+      hand = Hand.new
+      card1 = Card.new('C', 4)
+      card2 = Card.new('H', 4)
+      card3 = Card.new('D', 2)
+      card4 = Card.new('S', 2)
+      hand.add_card(card1)
+      hand.add_card(card2)
+      hand.add_card(card3)
+      hand.add_card(card4)
+      result = hand.value_hash
+      value_hash = { 4 => 2, 2 => 2 }
+      expect(result).to eq(value_hash)
+    end
+  end
```

Comment:
I see you test-drove the #value_hash method.. nice! Now that we know it works though, we don't really need these extra tests. Testing helper methods is usually considered overkill so long as the method they help is fully tested. The #contains_pair tests are enough to show us that it works as expected.